x-pack/plugins/monitoring/public/components/chart/chart_target.js

```diff
@@ -54,6 +54,34 @@ export class ChartTarget extends React.Component {
       xaxisOptions.min = _.get(timeRange, 'min');
       xaxisOptions.max = _.get(timeRange, 'max');
 
+      /* obtain lowest and highest values from series data to calculate
+       * fudge factor*/
+      let lowestValue = Number.MAX_SAFE_INTEGER;
+      let highestValue = 0;
+      let hasData = false;
+
+      for (const { data } of series) {
+        for (const item of data) {
+          if (item[1] === null) {
+            continue;
+          }
+          hasData = true;
+
+          if (item[1] < lowestValue) {
+            lowestValue = item[1];
+          }
+          if (item[1] > highestValue) {
+            highestValue = item[1];
+          }
+        }
+      }
+
+      if (hasData) {
+        const fudgeFactor = (highestValue - lowestValue) / 2;
```

Comment:
If the two values happen to be exactly equal or, possibly worse, very nearly equal, then our fudge factor may be too fine grained (`0` in the case of exactly equal).

I'm not completely sure about what solution we should have for this scenario, especially because defining "nearly equal" is a bit arbitrary when you consider number formatting for things like bytes (where `3.1 GB` and `3.1 GB` may be very different at the `MB` level).

We could possibly come up with a more methodical fudge factor:

1. If the difference is 0, then maybe we use a percentage of the value (any idea what percentage?).
2. If the difference is less than some minimum percentage (probably the same answer for 1), then we use that as a minimum.
3. We use the fudge factor.

@chrisronline / @pohzipohzi what do you think?

Reply:
I leave this to @chrisronline :)

Reply:
Hey @pohzipohzi 

I'm not sure when I'll have a chance to look at this again. Do you want to maybe make these changes and throw them up in the PR?